app/boards/intel_adsp_cavs25.conf

```diff
@@ -8,3 +8,7 @@ CONFIG_INTEL_ALH=y
 CONFIG_LP_MEMORY_BANKS=1
 CONFIG_HP_MEMORY_BANKS=30
 CONFIG_RIMAGE_SIGNING_SCHEMA="tgl"
+
+CONFIG_DEBUG_COREDUMP=y
+CONFIG_DEBUG_COREDUMP_BACKEND_INTEL_ADSP_MEM_WINDOW=y
+CONFIG_DEBUG_COREDUMP_MEMORY_DUMP_MIN=y
```

Comment:
debug feature should set to debug only build, release build should not have.
maybe we should have another set of config for debug build only.

Reply:
Coredump only happens with FW panic and it doesn't affect performance. It is used for debugging FW panic not for debug build

Reply:
can I understand that all the work will be done from zephyr side, and sof itself does not have code related with core dump?

Reply:
This is intented for release builds. We will not have logs enabled in release builds, so this coredump capability is the only way to get coredumps into kernel logs. Similar functionality has been enabled in all past (IPC3 based) SOF releases.

Reply:
> can I understand that all the work will be done from zephyr side, and sof itself does not have code related with core dump?

Most part of work is done in zephyr. SOF will only send a notification IPC message to host.